system/modules/isotope_reports/templates/iso_report_members_guests.html5

```diff
@@ -1,100 +1,117 @@
 <?php $this->insert('iso_block_panel', $this->arrData); ?>
 
 <div id="tl_buttons">
-<?php echo implode(' &nbsp; :: &nbsp ', $this->buttons); ?>
+  <?php echo implode(' &nbsp; :: &nbsp ', $this->buttons); ?>
 </div>
 
 <h2 class="sub_headline"><?= $this->headline ?></h2>
 
 <?= \Contao\Message::generate(true) ?>
 
 <div class="tl_listing_container list_view mod_iso_reports <?= $this->class ?>">
+  <?php $GLOBALS['TL_CSS'][] = 'system/modules/isotope_reports/assets/apexcharts/apexcharts.css'; ?>
+  <figure style="width:100%; height:300px" id="report-data"></figure>
+  <script src="<?= \Haste\Util\Debug::uncompressedFile('system/modules/isotope_reports/assets/apexcharts/apexcharts.min.js') ?>"></script>
```

Comment:
I think we can safely stop using `Debug::uncompressedFile` here, the feature is not included in Haste 5 anyway 😅